views/error.html

```diff
@@ -1,6 +1,7 @@
-
-<h1>
-	{{title}}
-</h1>
-
-<p>{{error.message}}</p>
+<div class="o-registry-ui__error">
+	<h1>
+		{{title}}
+	</h1>
+	<p>It seems like you're looking for the component: <br><span>{{trim '/components/' requestUrl}}</span></p>
```

Comment:
The error isn't _always_ a `404`, so this should probably be in a conditional. You can check the `error.status` property and if it's different than `404` then just display the message.